### ISSUE TITLE
NVDA screen reader is not announcing the selection of v-select once a selection has been made 

### ISSUE THREAD

**Issue by JackLeiLing** [vuetifyjs/vuetify#15032]
When a screen reader user has made a selection from the v-select dropdowns, the item they have selected is not announced, and the incorrect element role is announced. Example: “<Label>, edit select. Blank.” Or, when a screen reader user uses arrow keys to navigate through the menu items without expanding the list items, nothing is announced.

I have been trying to use slot to fix the issue without success. Looks like it is caused by the hidden input without a value after the v-select__selection div, please see the attached screenshot for more details.

![Screen Shot 2022-05-04 at 9 03 30 am](https://user-images.githubusercontent.com/21034104/166588202-45939a67-a945-4257-80bc-bfc9d44c9bd3.png)
